spec/beaker-hostgenerator/generator_spec.rb

```diff
@@ -7,15 +7,7 @@
 
 module BeakerHostGenerator
       describe Generator do
-        let(:options) {
-          {
-            list_platforms_and_roles: false,
-            disable_default_role: false,
-            disable_role_config: false,
-            hypervisor: 'vmpooler',
-          }
-        }
-        let(:generator) { BeakerHostGenerator::Generator.new(options) }
+        let(:generator) { BeakerHostGenerator::Generator.new }
```

Comment:
The `options` map is no longer 1-to-1 with the Generator, since it meant that the hypervisor was 1-to-1 as well.
The `Generator` no longer takes any arguments in the constructor, and instead the `options` are passed to the `generate` method directly.